app/dashboard/models.py

```diff
@@ -1121,6 +1121,17 @@ def is_staff(self):
         """
         return self.user.is_staff if self.user else False
 
+    @property
+    def is_funder(self):
```

Comment:
You can determine whether or not the user is classified as a `funder` by checking if `profile.stats[0][0] == 'funder'`

Reply:
> Ideally, we should split off this role determination to it's own property/method.
Do you mean this should not be a property on profile as we're currently doing?

Reply:
> How do I get an instance of the profile in my get method?

`self` ?

> Do you mean this should not be a property on profile as we're currently doing?

i believe he means you shouldnt have to incur the performance over head of `.stats` in order to learn if the user is a funder or hunter